timApp/i18n/messages.fi.xlf

```diff
@@ -6862,6 +6862,62 @@
+        <target state="translated"><x id="START_PARAGRAPH" ctype="x-p" equiv-text="&lt;p>"/> Viestien toimituksen varmistaminen on pois käytöstä. Kaikki listan asetukset voi vaihtaa, mutta jotkut niistä voivat auheuttaa viestien merkitsemisen roskapostiksi. <x id="CLOSE_PARAGRAPH" ctype="x-p" equiv-text="&lt;/p>"/><x id="START_PARAGRAPH" ctype="x-p" equiv-text="&lt;p>"/><x id="START_TAG_STRONG" ctype="x-strong" equiv-text="&lt;strong>"/>Tätä vaihtoehtoa ei suositella käytettäväksi!<x id="CLOSE_TAG_STRONG" ctype="x-strong" equiv-text="&lt;/strong>"/> Useimmat sähköpostipalveluntarjoajat merkitsevät listan kautta lähetetyt viestit todennäköisesti roskapostiksi. <x id="CLOSE_PARAGRAPH" ctype="x-p" equiv-text="&lt;/p>"/></target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">static/scripts/tim/messaging/manage/message-list-admin.component.ts</context>
+          <context context-type="linenumber">139,146</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="5842455841662368559" datatype="html">
+        <source><x id="START_PARAGRAPH" ctype="x-p" equiv-text="&lt;p>"/> The message list forwards messages to the receiver without modifications. <x id="START_TAG_STRONG" ctype="x-strong" equiv-text="&lt;strong>"/>This verification method disables some message list options.<x id="CLOSE_TAG_STRONG" ctype="x-strong" equiv-text="&lt;/strong>"/><x id="CLOSE_PARAGRAPH" ctype="x-p" equiv-text="&lt;/p>"/><x id="START_PARAGRAPH" ctype="x-p" equiv-text="&lt;p>"/> This option relies on the verification information provided by the sender's mail provider. <x id="CLOSE_PARAGRAPH" ctype="x-p" equiv-text="&lt;/p>"/></source>
+        <target state="translated"><x id="START_PARAGRAPH" ctype="x-p" equiv-text="&lt;p>"/> Viestilista välittää viestit vastaanottajalle ilman muutoksia.<x id="START_TAG_STRONG" ctype="x-strong" equiv-text="&lt;strong>"/>Tämä varmistustapa poistaa käytöstä joitakin viestilistan asetuksia.<x id="CLOSE_TAG_STRONG" ctype="x-strong" equiv-text="&lt;/strong>"/><x id="CLOSE_PARAGRAPH" ctype="x-p" equiv-text="&lt;/p>"/><x id="START_PARAGRAPH" ctype="x-p" equiv-text="&lt;p>"/>Tämä vaihtoehto perustuu lähettäjän sähköpostipalveluntarjoajan antamiin tarkistustietoihin.<x id="CLOSE_PARAGRAPH" ctype="x-p" equiv-text="&lt;/p>"/></target>
```

Comment:
Oliko jossain selvennetty mitä kaikkia asetuksia poistuu käytöstä?

Reply:
Ei ole, tällä hetkellä se on vain Subject prefix (menee disabled-tilaan, kun tämä vaihtoehto on valittu). Minäpä vaikka lisään tuohon selitykseen vielä tarkennuksen sen varalta, että tulevaisuudessa näitä asetuksia tulee lisää.

Reply:
Nyt on tarkempi selitys